docs/ota-updates.md

```diff
@@ -0,0 +1,77 @@
+# Over The Air Unattended Updates (OTAU2)
+
+This document enumerates some of the approaches and FLOSS software that could
+be used to deploy OTAU2 to Lepidopter distribution.
+
+## Requirements
+
+A list of Lepidopter's OTAU2 requirements:
+
+* Atomic software release update
+
+* On failure, deploy previous working bootloader, kernel
+    configuration, and filesystems
+
+* On success, deploy newest working bootloader, kernel
+    configuration, filesystems and reboot (if needed) for the changes to take
+    effect
+
+* Update of bootloader, kernel and configuration data, and filesystems
+
+* Support for signing of images and verification of images on
+    installation
+
+* Support for a self-hosted deployment server
```

Comment:
@bassosimone good point, we can perhaps use a cloud-fronted server were we 'll hand the updated images.